h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -3,27 +3,52 @@ import { Button, Input, Textarea } from '@hypothesis/frontend-shared';
 export default function CreateGroupForm() {
   return (
     <>
-      <h1>Create a new private group</h1>
+      <h1 class="mt-[55px] mb-[30px] text-[#3f3f3f] text-[19px] leading-[15px]">
+        Create a new private group
+      </h1>
 
       <form>
-        <div>
-          <label for="name">Name*</label>
+        <div class="mb-[15px]">
+          <label for="name" class="text-[#7a7a7a] text-[13px] leading-[15px]">
+            Name<span class="text-[#d00032]">*</span>
+          </label>
```

Comment:
The sequence of classes `text-[#7a7a7a] text-[13px] leading-[15px]` is repeated a few times, and the `text-[#d00032]` (for the `*`) is repeated a couple of times. This probably suggests that there's a sub-component I should factor out here. The sub-component may not be `<Label>` though since this CSS classes is also used for the character counters and for the `<footer>` at the bottom of the form neither of which are `<label>`'s